README.md

```diff
@@ -1 +1,61 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+> Setting up first "hello miceroverse" project.
+
+![image](https://user-images.githubusercontent.com/12495712/148043597-c6a72abe-64a9-4e2b-b776-6393348fac79.png)
+
+This project objective was to set up "hello microverse" project 
+
+## Built With
+
+- HTML and CSS
+- VSCode, GitHub
+
+## Live Demo
+
+[Live Demo Link](https://diwakarstha.github.io/hello-microverse/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+git clone https://github.com/diwakarstha/hello-microverse.git
+
+### Usage
+open index.html file
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@diwakarstha](https://github.com/diwakarstha)
+- Twitter: [@ShresthaDiwaka1](https://twitter.com/ShresthaDiwaka1)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/diwakarstha)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] This link is broken, please consider adding MIT.md license in your project directory to make the link work 👍🏻